src/renderers/shared/stack/reconciler/ReactCompositeComponent.js

```diff
@@ -362,9 +338,20 @@ var ReactCompositeComponentMixin = {
       markup = this.performInitialMount(renderedElement, hostParent, hostContainerInfo, transaction, context);
     }
 
+    var debugID = 0;
```

Comment:
Mostly consistency with other places where I pass it even in prod (as `0`), see `performInitialMount`. Passing `0` in that case doesn’t really make any difference, and is easier than to branching calls on `__DEV__`. I chose to pass `0` over passing `undefined` in production because I felt that passing different argument types depending on DEV/PROD several levels deep is potentially asking for trouble / misinterpretation.